src/components/Shared/CrossingMapPage/CrossingMapSidebar.js

```diff
@@ -154,117 +154,119 @@ class CrossingMapSidebar extends Component {
-                        <FilterCheckbox
-                          isChecked={showOpen}
-                          onClick={toggleShowOpen}
-                        >
-                          Open
-                        </FilterCheckbox>
-                        <FilterCheckbox
-                          isChecked={showClosed}
-                          onClick={toggleShowClosed}
-                        >
-                          Closed
-                        </FilterCheckbox>
-                        <FilterCheckbox
-                          isChecked={showCaution}
-                          onClick={toggleShowCaution}
-                        >
-                          Caution
-                        </FilterCheckbox>
-                        <FilterCheckbox
-                          isChecked={showLongterm}
-                          onClick={toggleShowLongterm}
-                        >
-                          Long-Term Closure
-                        </FilterCheckbox>
-                      </div>
-                    )}
-                  </div>
-                )}
-                {selectedCrossingId && (
-                  <div className="CrossingMapPage_sidebar-nearby-history-toggle">
+              {!searchFocused && selectedCrossingId && (
+                <SelectedCrossingContainer
+                  crossingId={selectedCrossingId}
+                  currentUser={currentUser}
+                  selectCrossing={selectCrossing}
+                  allCommunities={allCommunities}
+                />
+              )}
+              {!searchFocused && !selectedCrossingId && (
+                <div>
+                  <div className="CrossingMapPage_sidebar-filter-sort-toggle-container">
                     <div
                       className={classnames(
-                        'CrossingMapPage_sidebar-nearby-tab',
+                        'CrossingMapPage_sidebar-filter-toggle',
                         {
-                          selected: this.state.showNearby,
+                          selected: this.state.showFilters,
                         },
                       )}
-                      onClick={() => this.toggleNearbyHistory('nearby')}
+                      onClick={this.toggleFilters}
                     >
-                      <FontAwesome name="map-marker" /> Nearby
+                      <div className="CrossingMapPage_sidebar-filter-toggle-text">
+                        {this.state.showFilters ? (
+                          <span className="CrossingMapPage_sidebar-filter-toggle-icon">
+                            <FontAwesome name="minus" ariaLabel="Hide" />
+                          </span>
+                        ) : (
+                          <span className="CrossingMapPage_sidebar-filter-toggle-icon">
+                            <FontAwesome name="plus" ariaLabel="Show" />
+                          </span>
```

Comment:
Since the wrapping span element is the same for both in this conditional, could you extract it from the conditional and let it only toggling on the FontAwesome component?